src/utils/WASDPlusControls.js

```diff
@@ -0,0 +1,207 @@
+import AFRAME from "aframe";
+import * as THREE from "three";
+let bind = AFRAME.utils.bind;
+let shouldCaptureKeyEvent = AFRAME.utils.shouldCaptureKeyEvent;
+
+let KEYCODE_TO_CODE = {
+    "38": "ArrowUp",
+    "37": "ArrowLeft",
+    "40": "ArrowDown",
+    "39": "ArrowRight",
+    "87": "KeyW",
+    "65": "KeyA",
+    "83": "KeyS",
+    "68": "KeyD",
+    "32": "Space",
+    "16": "ShiftLeft"
```

Comment:
Should this have ShiftRight?

Reply:
So this object, which was a direct copy from A-Frame's wasd controls component, appears to only be used for backwards compatibility with the KeyboardEvent.keyCode value which has since been deprecated by Mozilla. (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode). I'm thinking it's best to just drop this object altogether, actually.